modules/installation-registry-storage-block-recreate-rollout.adoc

```diff
@@ -32,5 +32,76 @@ $ oc patch config.imageregistry.operator.openshift.io/cluster --type=merge -p '{
 ----
 +
 . Provision the PV for the block storage device, and create a PVC for that volume. The requested block volume uses the ReadWriteOnce (RWO) access mode.
+.. . You can define a VMware vSphere PersistentVolumeClaim by creating a file, `pvc.yaml`, with the following contents:
++
+[source,yaml]
+----
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: image-registry-storage <1>
+spec:
+  accessModes:
+  - ReadWriteOnce <2>
+  resources:
+    requests:
+      storage: 100Gi <3>
+----
+<1> A unique name that represents the PersistentVolumeClaim.
+<2> The PersistentVolumeClaim's access mode. With `ReadWriteOnce`, the volume can be mounted with read and write permissions by a single node.
+<3> The size of the PersistentVolumeClaim.
+
+.. Create the PersistentVolumeClaim from the file:
++
+[source,terminal]
+----
+$ oc create -f pvc.yaml
+----
+
 +
 . Edit the registry configuration so that it references the correct PVC.
+
+.. Create the PVC:
++
+[source,terminal]
+----
+$ oc create -n openshift-image-registry -f -
+----
++
+[source,yaml]
+----
+apiVersion: "v1"
+kind: "PersistentVolumeClaim"
+metadata:
+  name: "image-registry-pvc"
+spec:
+  accessModes:
+    - ReadWriteMany
+  resources:
+    requests:
+      storage: 100Gi
+  storageClassName: nfs01
+  volumeMode: Filesystem
+----
++
+.. Add the name of your PVC:
++
+[source,terminal]
+----
+$ oc edit configs.imageregistry.operator.openshift.io -o yaml
+----
++
+[source,yaml]
+----
+storage:
+  pvc:
+    claim: <1>
+----
+<1> Creating a custom PVC allows you to leave the `claim` field blank for default automatic creation of an `image-registry-storage` PVC.
++
+. Change `managementState` Image Registry Operator configuration from `Removed` to `Managed`. For example:
++
```

Comment:
I think it is no need to make this change. Another change is 
$ oc patch config.imageregistry.operator.openshift.io/cluster --type=merge -p '{"spec":{"rolloutStrategy":"**Recreate**","replicas":1}}'

Reply:
@wzheng1 I removed this step ("Change `managementState`") as you suggested. I did not add the `oc patch` command here, however, because it is already the first step in this procedure: https://github.com/openshift/openshift-docs/pull/25941/files#diff-a60b59b9a74c2fd8108510f1f83dbc8bd4a8272739df7e8bf09aa3a53da19570L31